lumen/ai/views.py

```diff
@@ -88,6 +88,9 @@ async def _render_component(self):
             return
 
         try:
+            if self.component is None or self.spec is None:
+                raise ValueError("No spec provided; please first fill out the Config tab.")
+
             if self._rendered:
```

Comment:
```suggestion
            if self._rendered and self.component is not None:
```

Reply:
Do we need this if `if self.component is None or...` is going to raise a ValueError?

Reply:
Oh I just saw your above message; this change doesn't work because it'll just go downstream.

<img width="649" alt="image" src="https://github.com/user-attachments/assets/fff48362-7d60-4fb4-b3b1-913a6f5475d0">

Reply:
<img width="604" alt="image" src="https://github.com/user-attachments/assets/e6fb9100-4ea7-4b26-a6eb-bfdfa25d460c">